chain/network/src/routing/graph_v2/mod.rs

```diff
@@ -634,6 +646,17 @@ impl GraphV2 {
         self.unreliable_peers.store(Arc::new(unreliable_peers));
     }
 
+    /// Logs the given batch of updates and the results from processing them.
+    fn write_event_logs(&self, updates: &Vec<NetworkTopologyChange>, oks: &Vec<bool>) {
+        for index in 0..updates.len() {
+            if oks[index] {
+                tracing::debug!(target: "routing", "Processed event {:?}", updates[index]);
+            } else {
+                tracing::debug!(target: "routing", "Rejected invalid distance vector {:?}", updates[index]);
+            }
+        }
+    }
```

Comment:
nit, optional suggestion, feel free to ignore :) 

I generally don't like pure logging functions that could panic if used improperly. (Even if in this case it's obvious it is being used properly, you never know what future refactors will do.)

If you want, you could rewrite this in a non-panicking way if you avoid the index. For example something with [zip()](https://doc.rust-lang.org/stable/std/iter/trait.Iterator.html#method.zip):

```suggestion
    fn write_event_logs(&self, updates: &Vec<NetworkTopologyChange>, oks: &Vec<bool>) {
        for (update, &ok) in updates.iter().zip(oks) {
            if ok {
                tracing::debug!(target: "routing", "Processed event {:?}", update);
            } else {
                tracing::debug!(target: "routing", "Rejected invalid distance vector {:?}", update);
            }
        }
    }
```

The difference is, if the lengths of the input vectors are not equal, this function will ignore the unmatched values in the iterator rather than panic with an index out of bound error.

Also, consider making it a static method without the `self` and call it as `Self::write_event_logs()` instead of `self.write_event_logs()`.